.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,6 +33,17 @@
 
 ## RSS or Atom Feed 
 
+* [ ] The content under feedsDotnet must be pure .NET meaning:
+
+* .NET itself
+* .NET libraries
+* .NET languages C#, F#, Visual Basic
+* Tools needed during development (for .NET)
+* CI/CD
+* SPA Frontend technologies like Angular, Vue.js, React etc.
+
+* [ ] the content under feedsOffTopic may contain off-topic posts or posts indirectly related to .NET like frontend-topics, database etc.
+
```

Comment:
Missing things like Azure services, Azure Hosting or any other topic thats totally related to .NET